target/linux/ipq40xx/image/generic.mk

```diff
@@ -549,6 +549,25 @@ define Device/ezviz_cs-w3-wd1200g-eup
 endef
 TARGET_DEVICES += ezviz_cs-w3-wd1200g-eup
 
+define Device/glinet_gl-a1300
+        $(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := GL.iNet
+	DEVICE_MODEL := GL-A1300
+	SOC := qcom-ipq4018
+	DEVICE_DTS_CONFIG := config@ap.dk01.1-c2
```

Comment:
Can't boot without this (default: config@1) ?

Reply:
Note: Now, ```config-1``` is used by default.

https://github.com/openwrt/openwrt/blob/3f814ecd2408da6ac96f741d3d6331ce93b79734/include/image-commands.mk#L295

Reply:
> Note: Now, `config-1` is used by default.

On QCA boards we still use config@1.
https://github.com/openwrt/openwrt/commit/969083634481c3ab5fb80509f385ef10ab45b55f

Reply:
Ah, I didn't noticed. Thanks!